doc/api/next_api_changes/deprecations/25588-KS.rst

```diff
@@ -0,0 +1,27 @@
+The parameter of ``Transform.transform_non_affine`` in subclasses are renamed to *values*
```

Comment:
I would perhaps just use a single title "Method parameters in subclasses renamed to match base classes" or similar, and then list these out with the old parameter names.